### PR TITLE
Add feature loadtopology

### PR DESCRIPTION
**Pull request by richiMarchi**
This PR adds two features to make the configuration of polycube more user friendly.

- a new /cupes/ api is created in polycubed, this allows to create multiple cubes at the time with a single request. This can be used to load the full configuration of a network topology.
- the CLI has been extended to support reading complex configurations from the command line and from files. For instance a full topology can be deployed by `polycubectl  cubes add < my_network.yaml`. This feature is available to all commands.

### CODE REVIEW

**Comment by mauriciovasquezbernal** [comment]
@richiMarchi could you please edit this PR adding a description of what is this about?

**Comment by acloudiator** [comment]
> @richiMarchi could you please edit this PR adding a description of what is this about?

Hi @richiMarchi 
Thanks for this PR, but please do add the description to comprehend the info without walking thru codes.

**Comment by frisso** [comment]
I feel this PR in fact enables to load commands and configurations from a file, not just the entire topology, but also individual commands; the description should be changed according.

File: Documentation/polycubectl/polycubectl.rst [hunk 1]
```diff
@@ -45,14 +43,68 @@ You can start the daemon typing ``sudo polycubed`` in another terminal. Refer to
...
+``polycubectl`` is service agnostic, hence the syntax is service dependent.
+However we can generalize the syntax as:
+
+::
+
+        polycubectl [parent] [command] [child] [argument0=value0] [argument1=value1]
+
+
+- ``parent``: path of the parent resource where the command has to be applied.
+- ``command``: ``add``, ``del``, ``show`` or ``set``.
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply Documentation/polycubectl/polycubectl.rst]: Good suggestion Matteo.

**Pull request merged**